SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+## Supported Versions
+
+These versions are currently receiving security updates.
+
+| Version      | Supported          | Notes |
+| ------------ | ------------------ | ----- |
+| 3.0.x (main) | :white_check_mark: | This is the next major version and has not yet been released. |
+| 2.8.x        | :white_check_mark: | This is the current version. |
```

Comment:
if we are going to say "This is the current version" then this PR should be opened on `2.8.X` branch, not main.

Otherwise we should rephrase this to something like "This is the latest released version" 🤔 🤷‍♂️

Reply:
Rephrased. I think we need something in this branch as its the default branch, but happy to also make a PR for 2.8 if you think its worth doing.

Reply:
Yeah, Ideally I'd love to not care about 2.8.X branch 😭 Some day 🤞